seatunnel-e2e/seatunnel-flink-connector-v2-e2e/src/test/resources/assertion/fakesource_to_assert.conf

```diff
@@ -25,64 +25,66 @@ env {
   #execution.checkpoint.data-uri = "hdfs://localhost:9000/checkpoint"
 }
 
+# Docoment-Link
+# https://seatunnel.apache.org/docs/category/source-v2
+# https://seatunnel.apache.org/docs/category/sink-v2
+# https://seatunnel.apache.org/docs/category/transform
+
 source {
-  # This is a example source plugin **only for test and demonstrate the feature source plugin**
-    FakeSource {
-      result_table_name = "fake"
-      field_name = "name,age"
+  FakeSource {
+    fields {
+      c_map = "map<string, string>"
+      c_array = "array<tinyint>"
+      c_string = string
+      c_boolean = boolean
+      c_tinyint = tinyint
+      c_smallint = smallint
+      c_int = int
+      c_bigint = bigint
+      c_float = float
+      c_double = double
+      c_decimal = "decimal(30, 8)"
+      c_null = "null"
+      c_bytes = bytes
+      c_date = date
+      c_time = time
+      c_timestamp = timestamp
     }
```

Comment:
For testcase just change the FakeSource fields definition:

from:
FakeSource {
       result_table_name = "fake"
       field_name = "name,age"
}

to:
FakeSource {
       result_table_name = "fake"
       fields {
         name = string
         age = int
       }
}

Reply:
thx，good idea.